projects/packages/forms/src/blocks/contact-form/class-contact-form-block.php

```diff
@@ -201,4 +220,24 @@ public static function load_view_scripts() {
 			)
 		);
 	}
+
+	/**
+	 * Check if the current user can view the block.
+	 * Every user can see it if the Contact Form module is active,
+	 * but if it is inactive, only admins can see it.
+	 *
+	 * This is only useful when the Contact Form package is used within the Jetpack plugin,
+	 * where the module logic exists.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return bool
+	 */
+	public static function can_manage_block() {
+		if ( ! class_exists( 'Jetpack' ) ) {
```

Comment:
This will make it harder to include the form block outside of the Jetpack plugin context. I would avoid this since we wanted this package not be dependent on Jetpack down the line. But its own stand alone plugin. 

Maybe this should be a filter that any plugins can overwrite?

Reply:
That class check should not block standalone plugins since it returns `true` for standalone plugins, but I'm happy to add a filter. Done in 06b9a9f2c6de57e68a6df327e1a10ba210f5c0d9

Ultimately, I'm afraid there will still be extra work before the package can be fully independent from the Jetpack plugin though.